### DISCUSSION TITLE
I accidentally signed as teacher

### DISCUSSION THREAD

**Discussion by keremcanseker** [community/community#51019] [GitHub Education] [answered]
Hi. Today I was signing in for github student but i choose myself as teacher by mistake how can i change this i am a student

**Comment by neutrinomuon** [comment, marked as answer]
Hi :)! I suggest that you first try contacting GitHub support or GitHub Education support to see if they can change the status for you. It's important to note that response times may vary due to high demand.

However, there is another possible solution that you could try (you can try to check first with GitHub support). You can reapply for the GitHub Student Developer Pack and select "Student" as your role by following these steps:

1. Go to the GitHub Education website at https://education.github.com/;
2. Click on the "Apply now" button located in the top right corner of the page.;
3. Choose "GitHub Student Developer Pack" and click "Apply now";
4. Sign in to your GitHub account (if you haven't already);
5. Scroll down to the "Verify your academic status" section and click "Learn more" under the "Students" section;
6. Follow the instructions to verify your student status;
7. After your student status has been verified, reapply for the GitHub Student Developer Pack and select "Student" as your role;

Please let me know if this solution works for you. I hope this helps! :)

**Reply by mmyrte** [reply]
I want to change the other way around - switching from student to teacher - but the application area is greyed out, so I cannot reapply with my new status as a teacher. I seem to recall that at one point, I could have started a second application process, but that seems no longer possible. Is this true?
